README.md

```diff
@@ -578,34 +578,14 @@ implementation 'com.squareup.moshi:moshi:1.6.0'
 Snapshots of the development version are available in [Sonatype's `snapshots` repository][snap].
 
 
-ProGuard
+R8 / ProGuard
 --------
 
-If you are using ProGuard you might need to add the following options:
-```
--dontwarn okio.**
--dontwarn javax.annotation.**
--keepclasseswithmembers class * {
-    @com.squareup.moshi.* <methods>;
-}
--keep @com.squareup.moshi.JsonQualifier interface *
-```
-Additional rules are needed if you are using Kotlin:
+If you are using R8 or ProGuard add the options from [this file](https://github.com/square/moshi/blob/master/moshi/src/main/resources/META-INF/proguard/moshi.pro).
```

Comment:
if you're using R8, you don't have to do anything, though, right? that's the benefit to bundling it into the jar.

Reply:
Yes. Once we ship it I plan to change all the READMEs to say ProGuard only and that for R8 you do nothing.

Reply:
@JakeWharton Is it still planned to update the README's for R8 case?